src/perfoutputwidget.h

```diff
@@ -0,0 +1,69 @@
+#ifndef PERFOUTPUTWIDGET_H
+#define PERFOUTPUTWIDGET_H
+
+#include <QWidget>
+
+namespace KParts {
+class ReadOnlyPart;
+}
+
+class QTemporaryFile;
+class QTextEdit;
+class QLineEdit;
+
+class PerfOutputWidget : public QWidget
```

Comment:
separate file, but also don't forget to add a .cpp with the implementation of the dot `PerfOutputWidget::~PerfOutputWidget() = default`, to make sure that the vtable isn't inlined